src/components/actionTag/ActionTag.tsx

```diff
@@ -0,0 +1,39 @@
+'use client'
+interface IPlusMinusTagProps {
+  children?: React.ReactNode
+  onClick?: () => void
+}
+interface IActionTagProps extends IPlusMinusTagProps {
+  color?: 'primary' | 'secondary'
+}
+
+const ActionTagColors = {
+  primary: 'bg-mint-6 text-white',
+  secondary: 'bg-gray-2 text-gray-7',
+}
+
+export default function ActionTag({ children = '', color = 'primary', onClick }: IActionTagProps) {
+  return (
+    <button
+      className={`${ActionTagColors[color]} body-M rounded-[99px] px-[11px] py-[5px]`}
+      onClick={onClick}
+    >
+      {children}
+    </button>
+  )
+}
+
+function Plus(props: IPlusMinusTagProps) {
+  return (
+    <ActionTag {...props} color="secondary">
+      + {props.children}
+    </ActionTag>
+  )
+}
+
+function Minus(props: IPlusMinusTagProps) {
+  return <ActionTag {...props}>- {props.children}</ActionTag>
+}
+
+ActionTag.Plus = Plus
+ActionTag.Minus = Minus
```

Comment:
```suggestion
'use client'
import { type PropsWithChildren } from 'react'

interface ActionTagProps {
  onClick?: () => void
  title?: string
  color: 'primary' | 'default'
}

const ActionTagColors = {
  primary: 'bg-mint-6 text-white',
  default: 'bg-gray-2 text-gray-7',
}

export const ActionTag = ({ children, color, onClick }: PropsWithChildren<ActionTagProps>) => {
  return (
    <button
      className={`${ActionTagColors[color]} body-M rounded-[99px] px-[11px] py-[5px]`}
      onClick={onClick}
    >
      {children}
    </button>
  )
}

export const Plus = ({ title, ...props }: ActionTagProps) => {
  return (
    <ActionTag {...props} color="primary">
      {/* svg 파일 추가 */}
      {title}
    </ActionTag>
  )
}

export const Minus = ({ title, ...props }: ActionTagProps) => {
  return (
    <ActionTag {...props} color="default">
      {/* svg 파일 추가 */}
      {title}
    </ActionTag>
  )
}

```

이렇게 코드 바꾸는건 어떤지 태윤님의 의견이 궁금합니다!